quickwit-metastore/test-data/file-backed-index/v0-376d2611b43771e119cae07bbf5dc343.expected.json

```diff
@@ -54,13 +54,10 @@
     "index_uri": "s3://quickwit-indexes/my-index",
     "indexing_settings": {
       "commit_timeout_secs": 301,
-      "demux_enabled": true,
```

Comment:
could be nice to keep one file with the old config to see if... backward compatibility is working.